lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -256,7 +272,20 @@
     }
 
     var strippedDefaultLocale = function () {
-        return platform.builtInRegexMatch(GetDefaultLocale(), /([^_]*).*/)[1];
+        if (isPlatformUsingICU) {
+            const def = platform.getDefaultLocale();
+            const match = platform.builtInRegexMatch(def, LANG_TAG_EXT_RE);
```

Comment:
>LANG_TAG_EXT_RE) [](start = 58, length = 16)

Ensure this regex is not undefined first. If it is undefined, run the initializer function function. This pattern is used elsewhere.

e.g.:
            if (!LANG_TAG_RE) {
                InitializeLangTagREs();
            }